CHANGELOG/CHANGELOG-1.3.md

```diff
@@ -26,7 +26,7 @@ Note that we start to track changes starting from v1.3.7.
 - Fix [open db file readonly mode for commands which shouldn't update the db file](https://github.com/etcd-io/bbolt/pull/365), see also [pull/292](https://github.com/etcd-io/bbolt/pull/292).
 
 ### Other
-- [Build bbolt CLI tool, test and format the source code using golang 1.17.13](https://github.com/etcd-io/bbolt/pull/297).
+- [Build bbolt CLI tool, test and format the source code using golang 1.19.7](https://github.com/etcd-io/bbolt/pull/409).
```

Comment:
Please revert this change for now, and do it when we bump go version for [release-1.3](https://github.com/etcd-io/bbolt/tree/release-1.3)

Reply:
Good point, sorry getting ahead of myself! 🤦🏻‍♂️